media-video/mpv/mpv-9999.ebuild

```diff
@@ -145,7 +145,8 @@ mpv_check_compiler() {
 		if { use opengl || use egl; } && ! tc-has-tls; then
 			die "Your compiler lacks C++11 TLS support. Use GCC>=4.8 or Clang>=3.3."
 		fi
-		if use vaapi && use cpu_flags_x86_sse4_1 && ! tc-is-gcc; then
+		if use vaapi && has_version '<media-video/ffmpeg-3.3:0' && \
+			use cpu_flags_x86_sse4_1 && ! tc-is-gcc; then
```

Comment:
1. Please add one tab more here so that we can easily distinguish code inside from line continuation of the condition.
2. Might be better to do `use` and `tc-is-gcc` checks before the `has_version` — they are certainly faster.